### DISCUSSION TITLE
libhackrf does not come with libhackrf-dev

### DISCUSSION THREAD

**Discussion by maichai** [Homebrew/discussions#1948] [Everyday usage]
There is a number of tools that use libhackrf-dev instead of libhackrf. Would it be possible to ship libhackrf-dev together with libhackrf in the future? is there an interims "hack" how get libhackrf-dev via brew?

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Adding-Software-to-Homebrew

**Comment by gromgit** [comment]
From a development POV, the `hackrf` bottle already contains:
* `libhackrf.h`
* `libhackrf.so`
* `libhackrf.pc` (so your project can use `pkg-config` to get the necessary CFLAGS, LDFLAGS and other build configuration stuff)

If there's still something missing, please open a PR that adjusts the build process accordingly: https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request
